apps/climatemappedafrica/src/components/Header/index.stories.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-anonymous-default-export */
+import React from "react";
+
+import Header from ".";
+
+export default {
+  title: "PesaYetu/Components/Header",
+};
+
+function Template({ title, ...args }) {
+  return <Header {...args}>{title}</Header>;
+}
+
+export const Default = Template.bind({});
+
+Default.args = {
+  overline: "Overline",
+  title: 'Title with <span class="highlight">Highlight</span>',
+  subtitle: "A short paragraph to describe what the header is all about.",
+};
```

Comment:
```suggestion
import React from "react";

import Header from ".";

const meta = {
  title: "PesaYetu/Components/Header",
};

function Template({ title, ...args }) {
  return <Header {...args}>{title}</Header>;
}

const Default = Template.bind({});

Default.args = {
  overline: "Overline",
  title: 'Title with <span class="highlight">Highlight</span>',
  subtitle: "A short paragraph to describe what the header is all about.",
};

export { Default };
export default meta;
```

Prefer fixing linting errors whenever possible instead of disabling them.